src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -57,277 +57,274 @@
-
-  "block.minecolonies.blockhutcitizen": "Citizen's Hut",
+  "minecolonies.config.canplayeruseresetcommand": "Can a Player use Reset Command",
+  "minecolonies.config.canplayeruseresetcommand.comment": "Should a player be allowed to use the /mc colony rs reset command? [Default: false]",
+  "block.minecolonies.blockhutcitizen": "Citizen Hut",
   "block.minecolonies.blockhutshepherd": "Shepherd's Hut",
-  "block.minecolonies.blockhutswineherder": "SwineHerders's Hut",
+  "block.minecolonies.blockhutswineherder": "Swine Herder's Hut",
   "block.minecolonies.blockhutcowboy": "Cowboy's Hut",
-  "block.minecolonies.blockhutchickenherder": "ChickenHerders's Hut",
+  "block.minecolonies.blockhutchickenherder": "Chicken Herder's Hut",
   "block.minecolonies.blockhutrabbithutch": "Rabbit Hutch",
-  "block.minecolonies.blockhutbeekeeper": "Beekeeper's Hut",
+  "block.minecolonies.blockhutbeekeeper": "Apiary",
   "block.minecolonies.blockhutbaker": "Bakery",
   "block.minecolonies.blockhutuniversity": "University",
-  "block.minecolonies.blockhutblacksmith": "Blacksmith",
+  "block.minecolonies.blockhutblacksmith": "Blacksmith's Hut",
   "block.minecolonies.blockhutbuilder": "Builder's Hut",
   "block.minecolonies.blockhutfarmer": "Farm",
   "block.minecolonies.blockhutlumberjack": "Lumberjack's Hut",
   "block.minecolonies.blockhutfisherman": "Fisherman's Hut",
   "block.minecolonies.blockhutminer": "Mine",
-  "block.minecolonies.blockhutstonemason": "Stonemason",
+  "block.minecolonies.blockhutstonemason": "Stonemason's Hut",
   "block.minecolonies.blockhuttownhall": "Town Hall",
-  "block.minecolonies.blocksubstitution": "Placeholderblock",
-  "block.minecolonies.blockbarrackstowersubstitution": "BarracksTower Placeholder",
-  "block.minecolonies.blockconstructiontape": "Construction tape",
-  "block.minecolonies.blockhutguardtower": "GuardTower",
+  "block.minecolonies.blocksubstitution": "Placeholder Block",
+  "block.minecolonies.blockbarrackstowersubstitution": "Barracks Tower Placeholder",
+  "block.minecolonies.blockconstructiontape": "Construction Tape",
+  "block.minecolonies.blockhutguardtower": "Guard Tower",
   "block.minecolonies.blockhutfield": "Field",
-
-  "block.minecolonies.blockhutcitizen.name": "Citizen's Hut",
+  "block.minecolonies.blockhutcitizen.name": "Citizen Hut",
   "block.minecolonies.blockhutshepherd.name": "Shepherd's Hut",
-  "block.minecolonies.blockhutswineherder.name": "SwineHerders's Hut",
+  "block.minecolonies.blockhutswineherder.name": "Swine Herder's Hut",
   "block.minecolonies.blockhutcowboy.name": "Cowboy's Hut",
-  "block.minecolonies.blockhutchickenherder.name": "ChickenHerders's Hut",
-  "block.minecolonies.blockhutrabbithutch.name": "Rabbit hutch",
-  "block.minecolonies.blockhutbeekeeper.name": "Beekeeper's Hut",
+  "block.minecolonies.blockhutchickenherder.name": "Chicken Herder's Hut",
```

Comment:
I'm sorry, this is bothering me. Chicken farmer seems like a better name than 'chicken herder'. You farm them, but you *don't* herd them.

Reply:
What about Swine Herder?

Reply:
Well, it's close, but I'd change it to Swineherd. The other alternative in the dictionary is swineman, but I much prefer swineherd.